acd-screen-share-test.js

```diff
@@ -0,0 +1,131 @@
+/* global describe, it, before, afterEach, after */
+import { PureCloudWebrtcSdk } from '../../dist/purecloud-webrtc-sdk';
+import { testUtils, callUtils } from 'purecloud-spigot';
```

Comment:
NIT: Maybe we should name it `genesysclout-spigot` before we get too far 😬

Reply:
that's a good idea. I always forget about the name change.